int32 startID
int32 count
string type
---
task[] result